### ISSUE TITLE
Add support for Friends of Hue Switches

### ISSUE THREAD

**Issue by philipparndt** [ZeroOne3010/yetanotherhueapi#22]
Currently, a Friends of Hue switch will be handled as an "unknown" sensor.
The problem is, that it is not possible to access the state of an unknown sensor.

Here is an example json message from the hue API for a "Senic" Friends of Hue Switch.
There seems to be nothing specific for "Senic" in this message.

```json
{
    "capabilities": {
        "certified": true,
        "inputs": [
            {
                "events": [
                    {
                        "buttonevent": 16,
                        "eventtype": "initial_press"
                    },
                    {
                        "buttonevent": 20,
                        "eventtype": "short_release"
                    }
                ],
                "repeatintervals": []
            },
            {
                "events": [
                    {
                        "buttonevent": 17,
                        "eventtype": "initial_press"
                    },
                    {
                        "buttonevent": 21,
                        "eventtype": "short_release"
                    }
                ],
                "repeatintervals": []
            },
            {
                "events": [
                    {
                        "buttonevent": 19,
                        "eventtype": "initial_press"
                    },
                    {
                        "buttonevent": 23,
                        "eventtype": "short_release"
                    }
                ],
                "repeatintervals": []
            },
            {
                "events": [
                    {
                        "buttonevent": 18,
                        "eventtype": "initial_press"
                    },
                    {
                        "buttonevent": 22,
                        "eventtype": "short_release"
                    }
                ],
                "repeatintervals": []
            }
        ],
        "primary": true
    },
    "config": {
        "on": true
    },
    "manufacturername": "PhilipsFoH",
    "modelid": "FOHSWITCH",
    "name": "Some Name",
    "productname": "Friends of Hue Switch",
    "recycle": null,
    "state": {
        "buttonevent": 22,
        "lastupdated": "2020-12-17T08:32:32"
    },
    "swupdate": {
        "lastinstall": null,
        "state": "notupdatable"
    },
    "swversion": null,
    "type": "ZGPSwitch",
    "uniqueid": "00:00:00:00:01:23:45:67-89"
}
```

Can you add support for this? Or would you accept a PR?

**Comment by ZeroOne3010** [comment]
Hey, thank you very much for this issue and the pull request! Sorry for the delay in replying. 🎅 I'd definitely love to add support for Friends of Hue. 👍 I'll comment some more on the PR.

**Comment by ZeroOne3010** [comment]
@philipparndt Would you mind checking whether my switch overhaul at #24 would make sense to you and would fulfil your need for FoH support?

**Comment by ZeroOne3010** [comment]
Resolved in #24.
